### PR TITLE
Update tools

### PR DESCRIPTION
**Pull request by donny-dont**
WebKitDev -> 0.5.1
git -> 2.39.0.1
python -> 3.11.1
pip -> 22.3.1
pywin32 -> 305
ruby -> 3.1.3-1
cmake -> 3.25.1
llvm -> 15.0.6

The updated WebkitDev fixes an issue where Install-Xampp was not reporting that the install wasn't successful. Downgrading xampp to the latest version that was successful.

xampp -> 7.4.29.1

### CODE REVIEW

**Comment by foopoiuyt** [comment]
We should validate that everything works through to the downstream playstation stuff mostly due to the python update since 3.x -> 3.y isn't necessarily guaranteed to be safe.

**Comment by donny-dont** [comment]
Yep will do a local build and test run before landing.

**Comment by foopoiuyt** [comment]
Wanted to comment because I think we may have deadlocked on this, were you waiting for me to approve before doing the final tests (as I was waiting for the tests before hitting approve)

**Review by foopoiuyt — APPROVED** [review]

**Pull request merged**
